src/modules/did/did.service.ts

```diff
@@ -124,23 +124,41 @@ export class DIDService implements OnModuleInit {
    * Also retrieves all claims from IPFS for the document.
    * @param {string} did
    */
-  public async addCachedDocument(did: string) {
-    const transaction = this.sentryTracingService.startTransaction(
-      'process-did',
-      'Process DID',
-      {
-        did,
-      }
-    );
+  public async addCachedDocument(did: string, isSync = false) {
```

Comment:
I am a bit confused here. I thought that `addCachedDocument` should be used only to add document in cache. Am I wrong?

Reply:
It was, right now it is also used in full mode sync since the steps are identically for this cases